app/views/tran_details/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<p style="color: green"></p>
+<h2>GROUP TRANSACTIONS</h2>
+<% @group_name = TranGroup.find(params[:tran_group_id]) %>
+<h5><strong>Group Name: </strong> <%= @group_name.groupName %></h5>
+<h5 style="color:skyblue"><strong>Group Total :</strong> <%= @total_value %></h5>
+<div id="tran_details">
+  <table class="table table-hover">
+    <thead>
+      <tr>
+        <th scope="col">Date</th>
+        <th scope="col">Transection Details</th>
+        <th scope="col">Amount</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @tran_details.each do |tran|%>
```

Comment:
- [ ] Kindly make sure that for a given group, the list of transactions is presented ordered by the most recent as requested in the requirements. The last trans created should be at the top

Reply:
resolved